streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBSessionStore.java

```diff
@@ -44,9 +43,12 @@
     public void init(final StateStoreContext context, final StateStore root) {
         super.init(context, root);
         this.stateStoreContext = context;
+
+        this.position = super.getPosition();
```

Comment:
Should we be checkpointing the position in this store as well?

Reply:
Wasn't sure since this is store is wraps a inner (segment) store.

Reply:
I see. While digging through the code, I noticed that it's actually `RocksDB{Session,Windowed}Store`, which wraps SegmentedBytesStore, which wraps the actual segments (RocksDBStores).

It turns out that SegmentedBytesStore is where we're tracking the position, and that's also where we do restoration and checkpointing, so in reality, we just don't need this logic in `RocksDB{Session,Window}Store` at all!